Core1Platform/UserInterface/01b_Password.md

```diff
@@ -0,0 +1,33 @@
+[!!Edit profile settings](../UsingCore1/02_EditProfileSettings.md)
+
+# Password
+
+*Any workspace > Avator icon > Menu entry Settings > Tab PASSWORD*
+
+![Password](../../Assets/Screenshots/Core1Platform/ProfileSettings/Password/Password.png "[Password]")
+
+**Change password**
+
+- [SAVE]   
+    Click this button to save the new password. The notice *Submitting data..* is displayed. If the entered password is not fulfilling the requirements for a valid password, an error message is displayed and you must edit the password.
+
+    
```

Comment:
If the entered password is valid, it is saved and the Password saved pop-up window is displayed.
Ich würde den Satz im Absatz oben tun und kein Screenshot (Password saved) dazu.

Reply:
da steht was völlig anderes